src/test/java/org/traccar/protocol/LaipacProtocolDecoderTest.java

```diff
@@ -22,7 +22,7 @@ public void testDecode() throws Exception {
         verifyNull(decoder, text(
                 "$AVSYS,MSG00002,14406,7046811160,64*1A"));
 
-        verifyNull(decoder, text(
+        verifyNotNull(decoder, text(
```

Comment:
Should we use `verifyAttributes` instead?

Reply:
Figured it out now that you mention it. Thanks.